appinventor/appengine/src/com/google/appinventor/client/StatusPanel.ui.xml

```diff
@@ -0,0 +1,10 @@
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
```

Comment:
Note for the UIBinder XML files we should include a copyright declaration. See NoProjectDialogBox.ui.xml for an example.